pbf/views.py

```diff
@@ -237,13 +258,11 @@ def add_player(request, game_id):
     if '-' in spirit_name:
         spirit_name, aspect = spirit_name.split(' - ')
     spirit = get_object_or_404(Spirit, name=spirit_name)
-    
-    if aspect in spirit_starting_energy.keys():
-        starting_energy = spirit_starting_energy[aspect]
-    else:
-        starting_energy = spirit_starting_energy[spirit.name]
-
-    gp = GamePlayer(game=game, spirit=spirit, color=colors[0], aspect=aspect, starting_energy=starting_energy)
+    spirit_and_aspect = spirit_name + ("/" + aspect if aspect else "")
```

Comment:
it's not lost on me that it's a little silly that a few lines above we split `spirit_name` (might contain a ` - `) into `spirit_name` (guaranteed not to contain `-`) and `aspect`, and then we recombine them into `spirit_and_aspect` here, whereas we could just use that original value of `spirit_name` which does have a hyphen.

I think the argument in favour of doing it the way it's done here is that all changes are in a contiguous series of instead of interspersed with existing code, but this argument only benefits short-term reviewability rather than long-term maintainability.

I do have some interest in changing this but right now I manually test each change and it'd be so great to be able to automate that.